oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -191,10 +191,10 @@ private static class RevisionsOptions extends Utils.NodeStoreOptions {
             fullGcProgressSize = parser.accepts("fullGcProgressSize", "The number of documents to check for " +
                             "garbage in each Full GC cycle")
                     .withRequiredArg().ofType(Integer.class).defaultsTo(10000);
-            fullGcMaxAge = parser.accepts("fullGcMaxAge", "The maximum age of the document in seconds " +
+            fullGcMaxAgeSec = parser.accepts("fullGcMaxAge", "The maximum age of the document in seconds " +
```

Comment:
I would keep both variable name & param name same.
```suggestion
            fullGcMaxAge = parser.accepts("fullGcMaxAge", "The maximum age of the document in seconds " +
```

Reply:
done